NeuralAmpModeler/NeuralAmpModeler.cpp

```diff
@@ -900,3 +917,23 @@ void NeuralAmpModeler::_UpdateMeters(sample** inputPointer, sample** outputPoint
   mInputSender.ProcessBlock(inputPointer, (int)nFrames, kCtrlTagInputMeter, nChansHack);
   mOutputSender.ProcessBlock(outputPointer, (int)nFrames, kCtrlTagOutputMeter, nChansHack);
 }
+
+void NeuralAmpModeler::_SetDisabledForAllControl(const bool disabled)
+{
+  if (const auto ui = GetUI(); ui != nullptr)
+  {
+    ui->GetControlWithParamIdx(kEQActive)->SetDisabled(disabled);
+    ui->GetControlWithParamIdx(kOutNorm)->SetDisabled(disabled);
+
+    ui->GetControlWithTag(kCtrlTagModelFileBrowser)->SetDisabled(disabled);
+    ui->GetControlWithTag(kCtrlTagIRFileBrowser)->SetDisabled(disabled);
+
+    ui->GetControlWithParamIdx(kNoiseGateActive)->SetDisabled(disabled);
+    ui->GetControlWithParamIdx(kNoiseGateThreshold)->SetDisabled(disabled);
+    ui->GetControlWithParamIdx(kInputLevel)->SetDisabled(disabled);
+    ui->GetControlWithParamIdx(kOutputLevel)->SetDisabled(disabled);
+    ui->GetControlWithParamIdx(kIRToggle)->SetDisabled(disabled);
+
+    ui->ForControlInGroup("EQ_KNOBS", [disabled](auto* ctrl) { ctrl->SetDisabled(disabled); });
```

Comment:
Does this get it right when you re-enable the plug-in but the EQ knobs are still disabled via `kEQActive`?

Reply:
After changes - yes ;)